### ISSUE TITLE
degit error with docker build

### ISSUE THREAD

**Issue by alancleary** [legumeinfo/gcv#417]
@adf-ncgr reports that degit throws an error when building with docker.
```console
$ git branch
* macro-metrics
   main
$
$ docker build .
#15 385.3 Error: Command failed: ./scripts/get-proto.js
#15 385.3
/gcv/dep/legumeinfo-microservices/node_modules/tiged/dist/index-e42359c2.js:442
#15 385.3             throw new DegitError(`could not download ${url}`, {
#15 385.3                   ^
#15 385.3
#15 385.3 DegitError: could not download
[https://github.com/legumeinfo/microservices/archive/fbd058cc467b655a11aa90add813f952a249c580.tar.gz
#15](https://github.com/legumeinfo/microservices/archive/fbd058cc467b655a11aa90add813f952a249c580.tar.gz#15) 385.3     at Degit._cloneWithTar
(/gcv/dep/legumeinfo-microservices/node_modules/tiged/dist/index-e42359c2.js:442:10)
#15 385.3     at processTicksAndRejections
(node:internal/process/task_queues:96:5)
#15 385.3     at async Degit.clone
(/gcv/dep/legumeinfo-microservices/node_modules/tiged/dist/index-e42359c2.js:237:4)
#15 385.3     at async Promise.all (index 2) {
#15 385.3   code: 'COULD_NOT_DOWNLOAD',
#15 385.3   url:
'https://github.com/legumeinfo/microservices/archive/fbd058cc467b655a11aa90add813f952a249c580.tar.gz',
#15 385.3   original: Error: getaddrinfo EAI_AGAIN [codeload.github.com](http://codeload.github.com/)
#15 385.3       at GetAddrInfoReqWrap.onlookup [as oncomplete]
(node:dns:72:26) {
#15 385.3     errno: -3001,
#15 385.3     code: 'EAI_AGAIN',
#15 385.3     syscall: 'getaddrinfo',
#15 385.3     hostname: '[codeload.github.com](http://codeload.github.com/)'
#15 385.3   }
#15 385.3 }
#15 385.3
```

**Comment by adf-ncgr** [comment]
to be honest, I have no memory of reporting this. If you can't reproduce it, we can probably chalk it up to flaky internet (or flaky adf-ncgr)

**Comment by alancleary** [comment]
I wasn't able to replicate this error. However, I've updated both the macro-metrics and main branches since you reported this, so I may have inadvertently fixed it.

I will note that the GCV Dockerfile uses a multistage build so you probably don't want to use `docker build .` in most case, but rather, `docker build --target dev .`, for example. See the [README](https://github.com/legumeinfo/gcv#docker) for details.

Closing.
